velox/vector/VariantToVector.h

```diff
@@ -32,7 +32,4 @@ VectorPtr variantToVector(
     const Variant& value,
     memory::MemoryPool* pool);
 
-// Convers a value at 'index' of 'vector' into a Variant.
```

Comment:
Do we still need variantToVector ? Can we remove the TODO?

// TODO Fold into BaseVector::createConstant API.